tests/conftest.py

```diff
@@ -77,4 +77,5 @@ def cleanup_db(db):
     yield
     schemas = db.get_schemas()
     for schema in schemas:
-        db.drop_schema(schema)
+        if schema not in ["guest", "executor"]:
```

Comment:
I don't think it's a good idea to hardcode the usernames. You could create an ENUM somewhere to keep all the names as enumerations.